src/test/java/io/supertokens/test/GetUsersByEmailQueryTest.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Not sure why this file exists.. please discuss with me how testing needs to be done. If you have a better method that is clearly much, much better, then we can go with that. Else, please stick to what we currently do.